aqua_stark/migrate.sh

```
@@ -0,0 +1,36 @@
+# Stop script on error
+set -e
+ 
+# Load environment variables from the appropriate file
+ENV_FILE=".env.sepolia"
+ 
+if [ -f "$ENV_FILE" ]; then
+  echo "Loading environment variables from $ENV_FILE..."
+  export $(grep -v '^#' "$ENV_FILE" | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)